docs/bot/commands/testing/test-leave.md

```
@@ -0,0 +1,268 @@
+* Ensures you're marked as present in database
+* Prevents data inconsistencies
+
+## Usage Example
+
+```bash
+/test-leave
+```
+
+**Response:**
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)